src/utils/types/common.ts

```diff
@@ -24,6 +26,10 @@ export interface SubmittableDialogProps extends DialogProps {
   onSubmit: () => void;
 }
 
+export type BackendReturn<T, U = []> =
```

Comment:
Ahh yes, generic my favorite

Reply:
I should probably ask, is this criticism or just an aside?

Reply:
just an aside